orchagent/bufferorch.cpp

```diff
@@ -827,7 +827,7 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
             return task_process_status::task_invalid_entry;
         }
 
-        if(tokens[0] == gMyHostName)
+        if((tokens[0] == gMyHostName) & (tokens[1] == gMyAsicName))
```

Comment:
Shouldn't this be - 
if((tokens[0] == gMyHostName) && (tokens[1] == gMyAsicName))